app/src/main/java/com/accountabilibuddies/accountabilibuddies/modal/Challenge.java

```diff
@@ -0,0 +1,106 @@
+package com.accountabilibuddies.accountabilibuddies.modal;
+
+import com.parse.ParseClassName;
+import com.parse.ParseObject;
+
+import java.util.Date;
+import java.util.List;
+
+@ParseClassName("Challenge")
+public class Challenge extends ParseObject {
+
+    //Default Parse columns
+    // objectId is a unique identifier for each saved object.
+    // createdAt and updatedAt represent the time that each object was created and last modified.
+    /**
+     Challenge {
+         long typeId -->Another table
+         String name;
+         String description;
+         String startDate;
+         String endDate;
+         int frequency; -->Another table
```

Comment:
Why are some of these primitives if they are foreign keys referencing other tables. I'm looking at type and frequency in particular. Shouldn't they be their own objects (e.g. `Type type`, `Frequency frequency`)?